lib/outputs.py

```diff
@@ -2,6 +2,7 @@
 
 
 def report(check_findings):
+    check_findings.sort(key=lambda x: x.region)
```

Comment:
Why this? To sort results by region name?

Reply:
Yes